### PR TITLE
Map to Localytics special profile IDs

### PR DESCRIPTION
**Pull request by brennan**
./gradlew build - successful locally
./gradlew test - successful locally

### CODE REVIEW

**Review by ladanazita — APPROVED** [review]
Looks good to me. One nit pick is the commit "Fixing tests" is misleading, as this commit adds new functionality, not just bumps powermock.

File: src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java [hunk 1]
```diff
@@ -134,16 +134,35 @@
       Localytics.setCustomerId(userId);
       logger.verbose("Localytics.setCustomerId(%s);", userId);
     }
+
     String email = traits.email();
     if (!isNullOrEmpty(email)) {
       Localytics.setIdentifier("email", email);
+      Localytics.setCustomerEmail(email);
       logger.verbose("Localytics.setIdentifier(\"email\", %s);", email);
+      logger.verbose("Localytics.setCustomerEmail(\"$email\", %s);", email);
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java]: Should be just `logger.verbose("Localytics.setCustomerEmail(%s);", email);`. We're just logging `Localytics.setCustomerEmail(email);`

File: src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java [hunk 2]
```diff
@@ -134,16 +134,35 @@
...
     }
+
     String name = traits.name();
     if (!isNullOrEmpty(name)) {
       Localytics.setIdentifier("customer_name", name);
+      Localytics.setCustomerFullName(name);
       logger.verbose("Localytics.setIdentifier(\"customer_name\", %s);", name);
+      logger.verbose("Localytics.setFullName(\"$full_name\", %s);", name);
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java]: Should be `logger.verbose("Localytics. setFullName(%s);", name);`

File: src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java [hunk 3]
```diff
@@ -134,16 +134,35 @@
...
+    }
+
+    String firstName = traits.firstName();
+    if (!isNullOrEmpty(firstName)) {
+      Localytics.setCustomerFirstName(firstName);
+      logger.verbose("Localytics.setCustomerFirstName(\"$first_name\", %s);", firstName);
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java]: Same issue as other logs.

File: src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java [hunk 4]
```diff
@@ -134,16 +134,35 @@
...
     }
+
+    String lastName = traits.lastName();
+    if (!isNullOrEmpty(lastName)) {
+      Localytics.setCustomerLastName(lastName);
+      logger.verbose("Localytics.setCustomerLastName(\"$first_name\", %s);", firstName);
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java]: Same issue as other logs.

**Review by f2prateek — APPROVED** [review]

**Pull request merged**